### PR TITLE
docs: Added PR template

### PR DESCRIPTION
**Pull request by ActuallyACat**
# Objective
We don't have a PR template. 

# Motivation and Context 
To help reviewers gain the context they need to appropriately review PRs, we should strive to include as much relevant information as possible. This template should act as a prompt for contributors to include information relevant to review.

# Screenshots (if appropriate)
N/A 

# Checklist
- [x] If this contains visual changes, has it been reviewed by a designer? 
- [x] Does there need to be right-to-left (RTL) options for localization and internationalization?
- [x] Have you built this component with Culture Amp's Web Accessibility guide in mind - [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility).

### CODE REVIEW

**Comment by okeeffed** [comment]
I feel much better about this one already! Here is the Performance UI one if you wanted any more templates to see what is around CA (not all is relevant):

## Objective
What is the purpose of this pr?

### Context
Why are we making these changes to the codebase?
Is there a ticket that you can link to?

### Causes
If you are fixing a bug, what was the root cause of the issue?

### Changes
Please detail any changes that you made.

### Notes
This is where you can note things like alternatives that you considered and disregarded, or anything particularly unusual or interesting about your implementation. 

### Verification
What sort of test coverage do we have for this change?
What areas are most important to focus on in QA?

### Checklist
* [ ] I have updated the test suite
* [ ] I have tested the change locally
* [ ] I have done the cross-browser testing [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)
* [ ] I am basing off the correct branch
* [ ] I have updated any relevant documentation or left useful comments in the code.
* [ ] I have reviewed the [security considerations](https://cultureamp.atlassian.net/wiki/spaces/AU/pages/923893881/Key+Security+Concepts) and highlighted them as appropriate.

**Comment by ActuallyACat** [comment]
I'm going to steal those checklist items, thanks @okeeffed :D

**Comment by sentience** [comment]
Although we’re trying to get away from this, is it worth suggesting contributors check that any stylesheet(s) they have changed are not shared by an Elm component for a React change, or vice versa?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Objective
+<!--- Describe your changes in detail -->
```
Comment (sentience) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: This heading and comment seem to say opposite things.

The objective might be "Improve accessibility of button".

Changes in detail might be "Add `tabindex=0` to `div`s that should receive keyboard focus."

If you want both, maybe something like:

> # What and how
>

**Comment by ActuallyACat** [comment]
> Although we’re trying to get away from this, is it worth suggesting contributors check that any stylesheet(s) they have changed are not shared by an Elm component for a React change, or vice versa?

This hasn't bitten us for a month or two now. Along a similar theme is the point that our visual regression testing is pretty underutilised, and will also help us catch those cases (among others!). I'll add an item for this instead.

**Review by okeeffed — APPROVED** [review]
Looks great to me with all the suggestions put in! I have no idea if I have approving power but you can still get a tick ✅

**Comment by mbylstra** [comment]
Another big 👍 from me. We had more discussion on this in https://cultureamp.slack.com/archives/GLZGHEKG9/p1601517139016300, particularly around whether items were requirements or suggestions and keeping it inline with the Contributing guidelines.

**Pull request merged**